machine/netberg/netberg_common_rangeley/kernel/driver-i2c-busses-ismt-set-bus_speed-100kHz.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/drivers/i2c/busses/i2c-ismt.c b/drivers/i2c/busses/i2c-ismt.c
+old mode 100644
+new mode 100755
+index 09a3e03..a82c962
+--- a/drivers/i2c/busses/i2c-ismt.c
++++ b/drivers/i2c/busses/i2c-ismt.c
+@@ -192,7 +192,11 @@ static DEFINE_PCI_DEVICE_TABLE(ismt_ids) = {
+ MODULE_DEVICE_TABLE(pci, ismt_ids);
+ 
+ /* Bus speed control bits for slow debuggers - refer to the docs for usage */
++#if 1 /* Foxconn: Set i2c bus_speed to 100kHZ */
```

Comment:
1. The "#if 1" will be removed.
2. Because the "CPLD" on Netberg's platform supports 100kHz bus speed only.